source/extensibility/command_palette.rst

```diff
@@ -12,48 +12,42 @@ Command Palette
 Overview
 ========
 
-The *command palette* bound to :kbd:`Ctrl+Shift+P` is an interactive list
-whose purpose is to execute commands. The command palette is fed by entries in
-``.sublime-commands`` files. Usually, commands that don't warrant creating a
-key binding of their own are good candidates for inclusion in a ``.sublime- commands``
-files.
+The *command palette* is an interactive list
+whose purpose is to execute commands.
+The command palette is fed entries
+by means of ``.sublime-commands`` files.
+Commands that don't warrant
+creating a key binding of their own
+are usually good candidates
+for inclusion in a ``.sublime- commands`` file.
 
-By default, the command palette includes many useful commands, and provides
-convenient access to individual settings as well as settings files.
+By default,
+the command palette includes many useful commands,
+and provides convenient access
+to individual settings as well as settings files.
 
 
-File Format (Commands Files)
-============================
+File Format of ``.sublime-commands`` Files

```

Comment:
Imo, the file format should probably go into the reference section entirely. But we'll talk about that in the restructure process.